tests/test_vb.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
-import unittest
-from bayes.vb import *
+import pytest
```

Comment:
just out of curiosity, why did you switch from unittest to pytest?

Reply:
It is just more verbose:
* no subclassing of `unittest.TestCase` saves some LOC
* python-esque checks like `assert a == b` instead of `self.assertEqual(a, b)`